setup.py

```diff
@@ -61,5 +61,6 @@
                     ('%s/bin' % RGW_INSTALL_PATH,
                     ['src/setup/rgw_setup', 'src/support/rgw_support_bundle']),
                     ('%s/mini-provisioner' % RGW_INSTALL_PATH,['VERSION']),
+                    ('%s/conf' % RGW_INSTALL_PATH,['conf/cortx_rgw.conf'])
```

Comment:
Yday we discussed to maintain cortx_rgw.conf in local pvc right, so this file (generated during rgw mini provisioning) is available if container goes down and newly booted container has access to it ? 
Confirm with Provisioning that     config: /etc/cortx is indeed a PVC

Reply:
No, This is template file, this will be available in /opt/seagate/cortx/rgw/conf  dir after RPM installation, During MP we are copying this file in 'config' path which is define in gconf(cluster.conf).
rgw config path in container - **/etc/cortx/rgw/<machin-id>/cortx_rgw.conf**